app.py

```diff
@@ -22,8 +25,8 @@
 GPIO.setmode(GPIO.BCM)  # Broadcom pin-numbering scheme
 GPIO.setup(fanPin, GPIO.OUT)  # LED pin set as output
 
-# Initial state for LEDs:
-GPIO.output(fanPin, GPIO.HIGH)
+# Initial of the relay
```

Comment:
Initial what of the relay? Apples? Oranges?

Reply:
outdated